content/en/platform/corda/4.11/community/release-notes.md

```diff
@@ -20,6 +20,12 @@ tags:
 
 # Corda Community Edition 4.11 release notes
 
+## Corda Community Edition 4.11.1 release notes
+
+Interoperability fix between 4.11 and pre-4.11 nodes when sending/fetching transactions for new data type: `TRANSACTION_RECOVERY`.
```

Comment:
For consistency can you use same format as Community release notes 4.10 https://docs.r3.com/en/platform/corda/4.10/community/release-notes.html

Reply:
Updated